app/views/mortgage_calculator/transaction_tax_calculator/show.html.erb

```diff
@@ -4,6 +4,17 @@
   </script>
 <% end %>
 
+<% if i18n_locale_namespace == 'land_and_buildings_transaction_tax' %>
+  <div class="global-alert global-alert--warning">
+    <div class="global-alert__content-container">
+      <p class="global-alert__message">
+        <%= raw I18n.t("#{i18n_locale_namespace}.holding_message_html",
+        link: 'https://www.revenue.scot/land-buildings-transaction-tax/tax-calculator/lbtt-property-transactions-calculator') %>
+      </p>
+    </div>
+  </div>
+<% end %>
```

Comment:
We can use a partial and Rails controller inheritance views here and remove this `if`:

Example:

```
   <%= render 'global_alert' rescue nil %>
```

**If you don't like the rescue nil** you can have empty template on app/views/mortgage_calculator/land_transaction_tax/_global_alert.html.erb.

Then on app/views/mortgage_calculator/land_and_buildings_transaction_tax/_global_alert.html.erb:

```
  <div class="global-alert global-alert--warning">
    <div class="global-alert__content-container">
      <p class="global-alert__message">
        <%= raw I18n.t("#{i18n_locale_namespace}.holding_message_html",
        link: 'https://www.revenue.scot/land-buildings-transaction-tax/tax-calculator/lbtt-property-transactions-calculator') %>
      </p>
    </div>
  </div>
```

What do you think?

Reply:
@tomas-stefano I think something along those lines is more elegant and removes the logic from the view, so will look at implementing something like that. 👍